float32[] order_serize
string obj_one
string obj_two
